src/test/java/com/lob/client/test/CheckTest.java

```diff
@@ -44,12 +44,9 @@ public void testListChecks() throws Exception {
         final CheckResponse response = responseList.get(0);
 
         assertTrue(response instanceof CheckResponse);
-        assertNotNull(response.getTracking().toString());
```

Comment:
Looks like we [lost some coverage](https://coveralls.io/builds/6916849) from removing these lines. We should add them back in the jobs tests so we don't lose that coverage.